requirements-dev.txt

```diff
@@ -0,0 +1,54 @@
+alabaster==0.7.11
```

Comment:
I'm wondering why we need this?

Reply:
I think might have gotten pulled in with one of the other dependencies? I can remove it when I figure out the conflicts.